### ISSUE TITLE
less 'gaps' in the documentation

### ISSUE THREAD

**Issue by techdragon** [jazzband/django-payments#46]
The documentation has some good examples of how to use the module, but it doesn't really show how to integrate it into existing django apps. 

Its easy to see some ambiguity in this documentation right here https://django-payments.readthedocs.org/en/latest/usage.html#making-a-payment

Where the code belongs isn't stated, its just an example of how to create an arbitrary payment, no explanation to novices where they should be putting that code, model method? view? signal? Redirect to the payment handling view? do i need to pass context? why / why not?

The installation page is closer to what the docs need.

I ask because it appears that your project is the only python 3 compatible django payments module that supports multiple payment providers, which feels worthy of some effort, ensuring people wanting python 3 django support also deserve well documented payment tools. 

Id write better docs myself but I don't really understand the library well enough. Without more docs, or some working examples to compare to, I'm not even sure I'm using the library correctly / as intended, and I don't want to write incorrect / unhelpful docs.

**Comment by litchfield** [comment]
Yes the docs are woefully incomplete- easier to read the code than try to understand the docs.

**Comment by MiltonLn** [comment]
Hello, i'm pretty new for payments in web apps and it's costing to me a lot to understand how to use this great app, there is some kind of example that i can follow? The documentation is not helping me too much, thank you :)

**Comment by patrys** [comment]
Please take a look at the Saleor repository for a full working store example.

**Comment by Astyan-42** [comment]
The documentation is very poor, but the module seems great. I will try to see how Saleor works with it. But, just in case, is there a smaller base of code using it available somewhere ?

**Comment by MacLake** [comment]
It would also be helpful to have some documentation on how to add a new payment method to django-payments.

**Comment by loudy-art** [comment]
Hi @techdragon did you sorted this out?  i mean did you implemented this on an existing project or created some kind of example project? because i'm having the same issue understanding the docs rn

**Comment by loudy-art** [comment]
hi @tohid4n , yes, i created [an example ](https://github.com/loudy-art/django-mercadopago-example) for people using MercadoPago which is latin america's most popular payment method. 

Here you can see my [views.py](https://github.com/loudy-art/django-mercadopago-example/blob/master/mp_site/pagos/views.py) which is highly commented so you can understand the payment flow that i used. In this case i was receiving form data and after payment was succesful i will send the form.
